app/controllers/orgs/lti_configurations_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Orgs
+  class LtiConfigurationsController < Orgs::Controller
+    before_action :ensure_lti_launch_flipper_is_enabled
+    before_action :ensure_current_lti_configuration, except: %i[new create]
```

Comment:
Does this controller support the `new` route? Is it convention to have a `#new` method even when it does nothing (like you did for `#create`)?

Reply:
The controller does.

That is a good question. Maybe we should move the information page to a different route and have the `#new` be where teachers actually create the new config. What do you think @thebradbain?

Reply:
`#new` as an action is kind of awkward in this flow because it essentially just displays a view with a button, that, once an instructor clicks "generate LMS config" will simply call out to the `#create` action and generate its `shared_secret` and `consumer_key` fields with no other user input, and then direct them to the #show action.

I could get behind `#index` instead, but as this is a page we would only show in the context of when an `LtiConfiguration` doesn't exist for the organization, I also wonder if that makes sense

Reply:
Yeah that makes sense. I'm ok with the info view being under `#new` function. It is a bit awkward but I think that's ok.

Reply:
The purpose of the `new` action in Rails is create an HTML form for creating a new resource ([src](https://guides.rubyonrails.org/routing.html#crud-verbs-and-actions)). I would say that a form containing only a button counts 👍  We are good to resolve this discussion.